### PR TITLE
Add viewing features for customer and supply orders in the frontend

### PR DESCRIPTION
**Pull request by samuelthen**

### CODE REVIEW

File: src/main/java/seedu/address/ui/CustomerOrderListPanel.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import javafx.scene.layout.Region;
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.model.order.CustomerOrder;
+
+import java.util.logging.Logger;
+
+/**
+ * Panel containing the list of persons.
+ */
+public class CustomerOrderListPanel extends UiPart<Region> {
```
Comment (calebchongsj) [inline src/main/java/seedu/address/ui/CustomerOrderListPanel.java:16]: Good to see that you have implemented the window panel for customer orders.

File: src/main/java/seedu/address/ui/SupplyOrderCard.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+    private static final String FXML = "SupplyOrderListCard.fxml";
+
+    /**
+     * Note: Certain keywords such as "location" and "resources" are reserved keywords in JavaFX.
+     * As a consequence, UI elements' variable names cannot be set to such keywords
+     * or an exception will be thrown by JavaFX during runtime.
+     *
+     * @see <a href="https://github.com/se-edu/addressbook-level4/issues/336">The issue on AddressBook level 4</a>
+     */
```
Comment (calebchongsj) [inline src/main/java/seedu/address/ui/SupplyOrderCard.java:23]: It's great that the supplier orders are integrated into a separate window panel from customer orders. With these two window panels, bakery owners can view both orders simultaneously.

**Review by calebchongsj — APPROVED** [review]
Great job to add a window panel view for customer and supply orders.

File: src/main/java/seedu/address/model/order/OrderList.java
Comment (ryenl) [inline src/main/java/seedu/address/model/order/OrderList.java:1]: Good use of ObservableList to reflect list changes in Ui

File: src/main/java/seedu/address/ui/CustomerOrderListPanel.java
Comment (ryenl) [inline src/main/java/seedu/address/ui/CustomerOrderListPanel.java:1]: Love the added Customer orders and Supplier orders panels which makes it easier to visualise pending orders

File: src/test/java/seedu/address/logic/commands/AddCommandTest.java
Comment (ryenl) [inline src/test/java/seedu/address/logic/commands/AddCommandTest.java:1]: Good use of testing stubs and assertions

**Review by ryenl — APPROVED** [review]
LGTM!

**Pull request merged**